app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -16,6 +17,26 @@ def allowed_helpers
       def errata(id)
         Katello::Erratum.in_repositories(Katello::Repository.readable).with_identifiers(id).map(&:attributes).first.slice!('created_at', 'updated_at')
       end
+
+      def host_subscriptions(host)
+        host.subscriptions
+      end
+
+      def host_applicable_errata_ids(host)
+        host.applicable_errata.map(&:errata_id)
+      end
+
+      def host_applicable_errata_filtered(host, filter = '')
+        host.applicable_errata.search_for(filter)
+      end
+
+      def host_latest_applicable_rpm_version(host, package)
+        host.applicable_rpms.where('name = ?', package).sort_by(&:version_sortable).first.try(:nvra)
```

Comment:
This is somewhat inefficient.  Why not `host.applicable_rpms.where('name = ?',  package).order(:version_sortable).limit(1).pluck(:nvra).first`?

Reply:
:+1: while at it, `where(name: package)` would also be nicer